libs/wingc/src/type_check.rs

```diff
@@ -203,13 +203,13 @@ pub enum Type {
 	Duration,
 	Boolean,
 	Void,
-	Json,
+	Json(Option<JsonData>),
 	MutJson,
```

Comment:
Why does this extra field exist on Json and not MutJson? (add comment)

Reply:
added